docker/Dockerfile

```diff
@@ -71,5 +76,5 @@ RUN \
 USER node
 ENV NODE_ENV=production
 
-ENTRYPOINT ["pm2-runtime", "bin/iotagent-json"]
+ENTRYPOINT ["docker/entrypoint.sh"]
```

Comment:
This file has no execute permissions `chmod +x docker/entrypoint.sh`

Reply:
Fixed by 46dac69bbf5bdff53bd1c1b03cc287c713fbe38c

Reply:
LGTM